### ISSUE TITLE
`prettier: 'compat'` still triggers `@html-eslint` style rules

### ISSUE THREAD

**Issue by fregante** [xojs/eslint-config-xo#106]
```js
export default defineConfig([
	includeIgnoreFile(fileURLToPath(new URL('.gitignore', import.meta.url))),
	...eslintConfigXo({
		browser: true,
		prettier: 'compat'
	}),
]);
```
```r
  110:1    error  Expected indentation of 11 tab but found 3 tab                                         @html-eslint/indent
  111:1    error  Expected indentation of 11 tab but found 3 tab                                         @html-eslint/indent
  111:6    error  There should be a linebreak after <p>                                                  @html-eslint/element-newline
  111:7    error  There should be a linebreak after </input>                                             @html-eslint/element-newline
```

This is due to:

- https://github.com/prettier/eslint-config-prettier/issues/248
